### PR TITLE
Add event service pyroscope config

### PR DESCRIPTION
**Pull request by boreq**

### CODE REVIEW

File: inventories/nos_event_service/inventory.yml [hunk 1]
```diff
@@ -9,6 +9,10 @@ nos_event_service:
     events_google_pubsub_credentials_json_path: '{{inventory_dir}}/group_vars/all/vault_pubsub-credentials.json'
     events_listen_address: 8008
     events_environment: PRODUCTION
+    events_pyroscope_application_name: events.nos.social
```
Comment (cooldracula) [inline inventories/nos_event_service/inventory.yml:12]: This isn't blocking for this PR or deployment, but I would generally recommend moving this variable down to the hosts specific section of the inventory file, or make it more general, like `events_pyroscope_application_name: "{{ inventory_hostname }}"`.    This set of variables apply to all hosts in the nos_event_service inventory group, and so is too hard-coded if we ever decided to have multiple event services running under different domains.

File: inventories/nos_event_service/inventory.yml [hunk 2]
```diff
@@ -9,6 +9,10 @@ nos_event_service:
...
+    events_pyroscope_server_address: https://pyroscope.nos.pink
```
Comment (cooldracula) [inline inventories/nos_event_service/inventory.yml:13]: Do we want to use a nos/planetary domain here?  If I remember correctly, nos.pink is a personal domain?  If I am misremembering, please ignore this comment!

**Review by cooldracula — APPROVED** [review]

**Comment by cooldracula** [comment]
This looks good to me!  I have a couple of comments, but they aren't blocking I don't think.  Variables are named well following our established pattern, and the work is straightforward.  The only issue is with the pyroscope application name variable.  It works well now, but could cause an issue in the future, setup-depending.

File: roles/nos-event-service/templates/env.tpl [hunk 3]
```diff
@@ -4,4 +4,7 @@ EVENTS_LOG_LEVEL={{ events_log_level }}
 EVENTS_DATABASE_PATH="/db/database.sqlite"
 EVENTS_GOOGLE_PUBSUB_PROJECT_ID="{{ events_google_pubsub_project_id }}"
 EVENTS_GOOGLE_PUBSUB_CREDENTIALS_JSON_PATH="/certs/{{ events_google_pubsub_credentials_json }}"
-EVENTS_PUBLIC_FACING_ADDRESS="https://{{ domain }}"
```
[the remark this answers is not in the document]
Comment (boreq) [reply roles/nos-event-service/templates/env.tpl:7]: It was never needed I most likely made a mistake documenting it somewhere as a configuration option when it never existed but I can't find that anywhere right now.

File: inventories/nos_event_service/inventory.yml (hunk 2 above)
Comment (boreq) [reply inventories/nos_event_service/inventory.yml:13]: Yeah I quickly spin up a server, I just need this for a couple of days for debugging purposes. If we decide to leave it running then we can move it to a different domain, most likely `planetary.tools`.

**Comment by boreq** [comment]
Sorry, I rebased this on top of `main` quickly without thinking about it instead of merging branches.

File: inventories/nos_event_service/inventory.yml (hunk 1 above)
Comment (cooldracula) [reply inventories/nos_event_service/inventory.yml]: Almost!  https://github.com/planetary-social/ansible-scripts/commit/c66f1efa8c39243c37295bd34b550d4bc577fbd2 sets it at the group level, and so all servers in the prod group would have the same app name, regardless of their domain.  We want to set it at [the host level](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#assigning-a-variable-to-one-machine-host-variables).  Again, this is a bit pedantic for this case, as there's only a single server.  It's hopefully helpful/interesting though.

I pushed a change(https://github.com/planetary-social/ansible-scripts/pull/65/commits/5324e03d7e0f536929aaba182c6bb55531c541bc) that sets that variable to specifically our events.nos.social server.

With that push, I think this is good to merge!

**Review by cooldracula — APPROVED** [review]

**Pull request merged**